### PR TITLE
Active job adapter (initialization fixed)

### PR DESCRIPTION
**Pull request by alachaum**
@vovimayhem See this PR. I've fixed the initialization logic by using a `initializer 'cloudtasker.active_job', after: :load_config_initializers`

Tested with and without eager loading + cache classes. Seems all good now.

### CODE REVIEW

**Review by vovimayhem — COMMENTED** [review]
Tested it some hours ago, but got caught up in the monday-at-the-office rush.

Works all OK, but I've got 2 issues:
- The changes in `lib/cloudtasker/cli.rb`, which may not belong to this PR (unless you think otherwise)
- What should we do with the original PR #13 ?

File: lib/cloudtasker/cli.rb [hunk 1]
```diff
@@ -64,6 +64,7 @@ def boot_system
       return false unless File.exist?('./config/environment.rb')
 
       require 'rails'
+      require 'cloudtasker/engine'
```
[the remark this answers is not in the document]
Comment (alachaum) [reply lib/cloudtasker/cli.rb:67]: This change is partially related. I had to fix the local server booting logic to be able to test the initialization logic with `cache_classes` and `eager_loading` locally. We can keep this change in this PR.

I think I'm going to rework a bit the local server at some point. I don't believe we need to load rails whatsoever.

**Comment by alachaum** [comment]
@vovimayhem I was planning to close your PR and merge this one. But if you prefer just cherry-pick my commits on your PR  and I'll merge yours. This way you'll retain original ownership of the PR.

**Comment by jpalley** [comment]
Hi @alachaum - great Gem!  Any plans to merge this PR in next couple of days?  We are deploying a new app and would like to use ActiveJob with cloudtaskers.  Thought it would be easiest/cleanest to test off your master.  Or we can branch and do our own merge to test.

**Comment by alachaum** [comment]
@jpalley yes, this PR is completely overdue. I'll go ahead and merge it in the next few hours then I will package a release candidate.

**Comment by alachaum** [comment]
@jpalley PR has been merged, the documentation has been updated and `0.11.rc2` has been released. Don't hesitate to let me know if you spot any issue with this RC.

Note: the extensions (cron, batch and unique jobs) are not available when using cloudtasker via ActiveJob because the latter does not provide the bindings we require (as far as I know).

**Pull request merged**